src/include/daos_types.h

```diff
@@ -132,8 +132,8 @@ typedef struct {
 	uuid_t			ci_uuid;
 	/** Epoch of latest persistent snapshot */
 	daos_epoch_t		ci_lsnapshot;
-	/** Redundancy factor */
-	uint32_t		ci_redun_fac;
+	/** Number of open handles */
+	uint32_t		ci_nhandles;
```

Comment:
why can't we have both (adding ci_nhandles at the end)?

Reply:
It would break the ABI that to my understanding cannot be done until the next major API version increment (that cannot be done in DAOS 2.4).

Reply:
you are already breaking the API, which IMO is actually worse

Reply:
Other than the daos_test are there other users of the ci_redun_fac structure member? Also in a previous commit I removed the other ci_redun_lvl member and we discussed the reasoning for making this subsequent change to ci_redun_fac as well https://github.com/daos-stack/daos/pull/9985#discussion_r952610355

This patch is a "nice to have" addition to what I've already implemented for daos_pool_filter_containers(). So, if we think that the API change (removing ci_redun_fac) is too severe, we can always just decide to abandon this patch (and give up the ability to track number of open handles). I think that would be a bad decision, since it seems this could be useful for a datamover use case (e.g., filter a pool's containers based on this and other criteria, to decide what data to migrate).

Reply:
"Other than the daos_test are there other users of the ci_redun_fac structure member?"
i don't know.. it's in a public header, so anyone can be using it. in the middleware i control / support, it's not used directly.
my comment is just in relation to your comment: "It would break the ABI"
you are already doing worse than that by breaking the API :-) so my point is that just adding the variables is not going to be worse that replacing an existing one.

Reply:
just to be clear, my change request is not for this. im fine if you want to just replace it here. hopefully no one is using it..
The main issue for this PR for me is the reported count in the num handles and not reporting the RF in the daos cont query (which seems unnecessary to remove in this case since you can just retrieve that from the open handle?)

Reply:
I do find it a bit odd to break the API on a minor API version change. I don't feel strongly enough to block on it, but I do hope it doesn't come back to bite us.

Reply:
Oh, if it was added recently, e.g. after branching 2.2, that's fine. I thought it was already in a release, which could mean users expect it and we shouldn't remove it. But if it's only in master, in active development, I think we don't have to worry about it.